app/views/appointments/none_available.html.haml

```diff
@@ -0,0 +1,22 @@
+.grid-row
+  %section#confirmation.column-two-thirds
+    %h1
+      Booking your appointment
+
+    %p
+      Sorry, we can't book an appointment for you online right now.
+
+    .box-highlight.lede
+      Your reference number is
+      = succeed "." do
+        %strong= @work_order_reference
+
+    %p
+      %strong
+        Please call us on
+        = repairs_contact_centre_telephone_number
+        to arrange an appointment
+      and quote your reference number
+      = succeed "." do
+        %strong= @work_order_reference
+      We're open Monday to Friday, 8am to midnight and Saturday, 9am to 1pm.
```

Comment:
We added the opening hours to the i18n file, because it changed a few times during the project, i.e:

```haml
%p
  We're open
  = succeed '.' do
    = t('rcc_opening_hours')
```

Probably worth doing this here as well.

Reply:
Oh, interesting. I copied these from the `_contact_phone_numbers` partial, so I should probably update that too!

Reply:
Done.